src/collections/_documentation/enriching-error-data/set-tag/go.md

```diff
@@ -1,5 +1,11 @@
 ```go
 sentry.ConfigureScope(func(scope *sentry.Scope) {
 	scope.SetTag("{{ page.example_tag_name }}", "{{ page.example_tag_value }}")
+
+    // to set multiple key-value pairs at once:
+    scope.SetTags(map[string]interface{}{
+        "{{ page.example_tag_name }}": "{{ page.example_tag_value }}",
```

Comment:
Indentation as well

Reply:
`SetTags` accept `map[string]string`, not `map[string]interface{}`

Reply:
✔️ on tabs. And ohhhh, right - of course. Pattern matched from `setExtra`, but that makes sense, since tags have to be strings (which `extra` values don't).

Reply:
Oh, damb - I bet that's going to apply to C#, too. Fixing...